contracts/contracts/lib/CrossMsgHelper.sol

```diff
@@ -199,4 +203,8 @@ library CrossMsgHelper {
 
         return true;
     }
+
+    function validateCrossMessage(IpcEnvelope memory crossMsg) internal view returns (CrossMessageValidationOutcome)  {
+        return LibGateway.validateCrossMessage(crossMsg);
```

Comment:
I still kind of feel `CrossMessageValidationOutcome` should not be exposed, unless different callers might handle the outcomes differently? Now every call of `validateCrossMessage` must handle the outcome, dont feel it's necessary though.

Reply:
`unless different callers might handle the outcomes differently? ` - that's exactly the case :)